connector/fyle_integrations_platform_connector/apis/expenses.py

```diff
@@ -176,6 +176,8 @@ def __construct_expenses_objects(self, expenses: List[dict]) -> List[dict]:
                     'cost_center': expense['cost_center']['name'] if expense['cost_center'] else None,
                     'corporate_card_id': expense['matched_corporate_card_transactions'][0]['corporate_card_id'] \
                         if expense['matched_corporate_card_transactions'] else None,
+                    'masked_corporate_card_number': expense['matched_corporate_card_transactions'][0]['masked_corporate_card_number'] \
```

Comment:
let's handle even key safely. 
`expense['matched_corporate_card_transactions'][0].get('masked_corporate_card_number')`

Reply:
I checked prod db, its present always if the `matched_corporate_card_transactions` is not empty. Still will handle this safely

Reply:
yes, we shouldn't trust a lot :P